tasks/variables.yml

```
@@ -52,6 +52,11 @@
     patroni_system_packages: "{{ __patroni_system_packages | list }}"
   when: patroni_system_packages is not defined
 
+- name: Append haproxy to system packages if needed
+  set_fact:
+    patroni_system_packages: "{{ patroni_system_packages + __patroni_haproxy_package | list }}"
```

Let's change the `__patroni_haproxy_package` name to `__patroni_haproxy_packages` for consistency with other `patroni_*_packages` variables.
```suggestion
    patroni_system_packages: "{{ patroni_system_packages + __patroni_haproxy_packages | list }}"
```

---

yup